### PR TITLE
added domain for GenerelSchwerz, homehosting

### PR DESCRIPTION
**Pull request by GenerelSchwerz**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

- [ ] You have completed your website. 
- [ ] The website is reachable.  
I am attempting to set up a domain for an SSH connection through Cloudfare Tunnels. See: [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/use-cases/ssh/#connect-to-ssh-server-with-cloudflared-access)

If I have set this up incorrectly or am not able to do this via the provided domain, please let me know! Thank you.

## Website Link/Preview
N/A

### CODE REVIEW

File: domains/homehosting.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"description": "Student hosting for SSH devices without open ports.",
+	"owner": {
+		"username": "GenerelSchwerz",
+		"email": "generelschwerz@gmail.com",
```
Comment (Stef-00012) [inline domains/homehosting.json:5]: remove the trailing comma
```suggestion
		"email": "generelschwerz@gmail.com"
```

File: domains/homehosting.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	},
+	"record": {
+		"A": [
+			"156.146.99.77"
```
Comment (Stef-00012) [inline domains/homehosting.json:9]: you need to put a website for the domain to be accepted

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/homehosting.json (hunk 2 above)
Comment (GenerelSchwerz) [reply domains/homehosting.json:9]: I cannot provide a website as I wish to use the domain provided here as the domain that cloudflare will redirect to. 

[Documentation for what I am attempting to do](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/use-cases/ssh/#connect-to-ssh-server-with-cloudflared-access)

Basically, I need a domain that can have its nameservers changed to cloudflare's. Once that's done, cloudflare will route everything to an IP that normally cannot have ports open. 

TL:DR; I cannot provide a service until a domain with modifiable nameservers is provided; after that is done, the service will be available. Is that possible with this service?

Comment (orxngc) [reply domains/homehosting.json:9]: > until a domain with modifiable nameservers is provided

We do not support NS records.

File: domains/homehosting.json
Comment (omsenjalia) [inline domains/homehosting.json:1]: This is not the issue

**Comment by EducatedSuddenBucket** [comment]
i think he wants to just use the domain for ssh

**Comment by orxngc** [comment]
> i think he wants to just use the domain for ssh

We don't allow this.
Closing stale (no user response within 48 hours.)
